controllers/azuremachine_controller.go

```diff
@@ -76,25 +72,28 @@ func NewAzureMachineReconciler(client client.Client, log logr.Logger, recorder r
 
 // SetupWithManager initializes this controller with a manager.
 func (amr *AzureMachineReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options Options) error {
-	ctx, _, done := tele.StartSpanWithLogger(ctx, "controllers.AzureMachineReconciler.SetupWithManager")
+	ctx, log, done := tele.StartSpanWithLogger(ctx,
+		"controllers.AzureMachineReconciler.SetupWithManager",
```

Comment:
clarification:
This logger previously had the name  created as`ctrl.Log.WithName("controllers").WithName("AzureMachine")`. Is this okay to change now? Same for other controllers as well.

Reply:
I would assume so, but it sounds like you are concerned about something specific. What concerns you about changing these?

Reply:
are we loosing info on the namespace and the name of the object? I see `logger.WithValues("namespace", req.Namespace, "azureMachine", req.Name)` was removed

Reply:
> are we loosing info on the namespace and the name of the object? I see logger.WithValues("namespace", req.Namespace, "azureMachine", req.Name) was removed

Indeed, that was removed, but it was added back in the tele.KVP attributes that are passed into `tele.StartSpanWithLogger`, which adds them as values to the composite logger.